units/root-fs-ready.target

```diff
@@ -0,0 +1,15 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Root File System is Ready
+Documentation=man:systemd.special(7)
+
+DefaultDependencies=no
+Before=local-fs.target
```

Comment:
This conflicts with the diagram from the commit message no? The diagram says root-fs-ready.target is ordered before sysinit.target but here we say its ordered before local-fs.target